cloudzero/README.md

```diff
@@ -0,0 +1,46 @@
+# Agent Check: CloudZero
+
+## Overview
+
+### Overview
+CloudZero helps engineering teams build cost-effective software. Through its innovative platform, CloudZero allocates 100% of customers' cloud, PaaS, and SaaS spend - regardless of their tagging quality - and presents it in a single unified view. It then combines hourly spend data with business- and system-level telemetry to put cloud data in a business context, through precise unit cost metrics like cost per customer, per product, per feature, per team, and more. CloudZero's AI-powered anomaly detection alerts engineers to abnormal spend events, pointing them directly to the affected infrastructure, and promoting engineering engagement in cloud cost management.
+
+### Benefits
+Once connected, the CloudZero platform regularly ingests your Datadog billing information for both committed and on-demand costs across all Datadog products. CloudZero unifies this data with the rest of your cloud costs, giving you a comprehensive assessment of your total cloud investment. The platform then puts 100% of your cloud spend through numerous analytics features, revealing opportunities for efficiency and letting you craft and send custom reports.
+
+## Setup
+
+### Installation
+
+Log in to your [CloudZero account](https://app.cloudzero.com) and navigate to the Connections settings. Create a new connection to Datadog. Once a connection is created, you will click "Authorize Datadog Account" to authorize CloudZero to pull data from your Datadog account into the CloudZero platform.
```

Comment:
As discussed, we highly recommend prompting the user to initiate authorization from Datadog. Can you add in that option as well?